pyproject.toml

```diff
@@ -76,6 +76,7 @@ testpaths = [
         "C0114", # mozilla-it/ctms-api#100
         "C0115", # mozilla-it/ctms-api#100
         "C0116", # mozilla-it/ctms-api#100
+        "C0121", # Because it's actually not a singleton, but a model comparison
```

Comment:
I'm not sure which like is giving this warning, but could just that line disable the warning?  The line syntax is ``# pylint: disable foo-message`` or ``# pylint: disable C0121``

Reply:
I've moved this inline; but it was being called out near the filtration query for the `==` when it believes singletons are being compared